### ISSUE TITLE
consider standard protocol specification like AsyncAPI

### ISSUE THREAD

**Issue by crotwell** [FDSN/SeedLink#6]
Perhaps worth considering something like 
https://www.asyncapi.com/docs
for standard way to specify the protocol.

I'm not sure if this is actually worth it, but probably should at least look into it as following a standard may be useful in generating api documentation, automated testing and interaction with tools, etc.

**Comment by andres-h** [comment]
### Feedback from proposal team

This may be a good idea, but we wouldn’t want to hold up the specification. It can be added later.

**Comment by crotwell** [comment]
Agree, propose we leave this issue open for future consideration, but not be part of v4 spec.
